apps/src/code-studio/components/progress/ResourcesDropdown.jsx

```diff
@@ -32,36 +32,60 @@ export default class ResourcesDropdown extends React.Component {
   handleDropdownClick = () => {
     if (this.props.unitGroupId) {
       this.recordFirehose('unit-group', 'click-dropdown', {
-        unitGroupId: this.props.unitGroupId
+        unitGroupId: this.props.unitGroupId,
+        studentFacing: !!this.props.studentFacing
       });
     } else if (this.props.unitId) {
       this.recordFirehose('unit', 'click-dropdown', {
-        unitId: this.props.unitId
+        unitId: this.props.unitId,
+        studentFacing: !!this.props.studentFacing
       });
     }
   };
 
-  handleItemClick = () => {
+  handleItemClick = (e, resource) => {
+    // Needed so that we can keep the href on the link to allow for standard link interactions
+    e.preventDefault();
+    const callback = () => {
+      window.open(resource.url, 'noopener', 'noreferrer');
+    };
+    const resourceKey = this.props.useMigratedResources
+      ? resource.key
+      : resource.type;
     if (this.props.unitGroupId) {
-      this.recordFirehose('unit-group', 'click-resource', {
-        unitGroupId: this.props.unitGroupId
-      });
+      this.recordFirehose(
+        'unit-group',
+        'click-resource',
```

Comment:
I know this name isn't new so probably nothing we can do here but would be great if we could more clearly differentiate these resources from the ones in the lesson plans